apps/site/src/lib/components/Button/Button.tsx

```diff
@@ -7,16 +7,17 @@ interface ButtonProps {
 	text: string;
 	className?: string;
 	href?: ComponentProps<typeof Link>["href"];
+  alt?: boolean;
```

Comment:
Suggestion: It might be better semantically to separate the two styles as "light" and "dark" versions, so we would replace `alt` with something like `useLightVersion` or something like that.